opencog/nlp/relex2logic/rule-helpers.scm

```diff
@@ -466,26 +466,26 @@
 	(define adv (cog-name adv-node))
 	(define adv_instance (cog-name adv-instance-node))
 	(ListLink
-		(InheritanceLink  (ConceptNode adv_instance) (ConceptNode adv))
-			(ImplicationLink  (PredicateNode instance) (PredicateNode verb))
-			(r2l-wordinst-Predicate instance)
-			(r2l-wordinst-concept adv_instance)
-			(InheritanceLink  (SatisfyingSetLink (PredicateNode instance)) (ConceptNode adv_instance))
+		(ImplicationLink  (PredicateNode adv_instance) (PredicateNode adv))
+		(InheritanceLink  (ConceptNode instance) (ConceptNode verb))
```

Comment:
Verbs should be predicates, nouns should be concepts...

On Wed, Jun 15, 2016 at 6:23 PM, Linas Vepštas <notifications@github.com>
wrote:

> In opencog/nlp/relex2logic/rule-helpers.scm
> <https://github.com/opencog/opencog/pull/2194#discussion_r67136700>:
>
> > @@ -466,26 +466,26 @@
> >  	(define adv (cog-name adv-node))
> >  	(define adv_instance (cog-name adv-instance-node))
> >  	(ListLink
> > -		(InheritanceLink  (ConceptNode adv_instance) (ConceptNode adv))
> > -			(ImplicationLink  (PredicateNode instance) (PredicateNode verb))
> > -			(r2l-wordinst-Predicate instance)
> > -			(r2l-wordinst-concept adv_instance)
> > -			(InheritanceLink  (SatisfyingSetLink (PredicateNode instance)) (ConceptNode adv_instance))
> > +		(ImplicationLink  (PredicateNode adv_instance) (PredicateNode adv))
> > +		(InheritanceLink  (ConceptNode instance) (ConceptNode verb))
>
> Since when are verbs "Concepts"? I thought only nouns are "concepts", and
> verbs are "predicates", right?
>
> Or maybe the R2L subsystem doesn't actually make this kind of distinction?
> Or maybe this distinction doesn't matter?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencog/opencog/pull/2194/files/c273c1c62c0728ab6d575ff7538a130e543ebef8#r67136700>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFolXNGc2MFihCBssjYmATboet5VEp2Mks5qL9KYgaJpZM4I1WZs>
> .
>



-- 
Ben Goertzel, PhD
http://goertzel.org

Super-benevolent super-intelligence is the thought the Global Brain is
currently struggling to form...